scripts/community_scoring.py

```diff
@@ -6,16 +6,23 @@
 the most influential communities that have the highest 
 potential of fraudulent activity. 
 '''
+import math
 import networkx as nx
-
+import pandas as pd
 
 def make_unfrozen_subgraph(G, community):
     sub_graph = G.subgraph(community)
     unfrozen_graph = nx.Graph(sub_graph)
+    for node, data in list(unfrozen_graph.nodes(data=True)):
+        if 'labels' in data and 'manifest' in data['labels']:
```

Comment:
should be 

if 'labels' in data and ('manifest' or 'destination') in data['labels']:

Reply:
or we can just remove because manifest is no longer in the database